src/py/flwr/client/clientapp/app.py

```diff
@@ -88,27 +97,23 @@ def on_channel_state_change(channel_connectivity: str) -> None:
 
 def run_clientapp(  # pylint: disable=R0914
     supernode: str,
+    run_once: bool,
     token: Optional[int] = None,
+    flwr_dir_: Optional[str] = None,
```

Comment:
Can we rename this to `flwr_dir`? We usually don't use trailing underscores in function parameters.